core/zoe/contracts/coveredCall.js

```diff
@@ -16,7 +16,7 @@ export const makeContract = harden((zoe, terms) => {
       offerHandle,
       offerRules: { payoutRules },
     } = await zoe.burnEscrowReceipt(escrowReceipt);
```

Comment:
For any file in which I still see mention of a receipt, I assume that this has not been converted to new style and so should not be reviewed?

Reply:
yup, that's correct. So far the covered call, autoswap, and myFirstDapp have not been changed